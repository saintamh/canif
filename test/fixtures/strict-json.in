[
{
"name": "A short-ish string",
"id": 1274430,
"withFinalComma": false
},
{
"name": "A somewhat longer but still short-ish string",
"id": 37.2,
"withFinalComma": true,
},
{
"name": "A string which is pretty long, not craazy long mind you, but, you know, long enough that it's unlikely to fit on a single line of pretty-printed JSON. Longer than is convenient, let's say",
"id": 5.37e-1,
}
]
