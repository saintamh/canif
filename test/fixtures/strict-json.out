[
    {
        "id": 1274430,
        "name": "A short-ish string",
        "withFinalComma": false
    },
    {
        "id": 37.2,
        "name": "A somewhat longer but still short-ish string",
        "withFinalComma": true
    },
    {
        "id": 0.537,
        "name": "A string which is pretty long, not craazy long mind you, but, you know, long enough that it's unlikely to fit on a single line of pretty-printed JSON. Longer than is convenient, let's say"
    }
]
